store_test.go

```diff
@@ -485,3 +489,84 @@ func TestWithSplitStore(t *testing.T) {
 	store.Free()
 	store.Free()
 }
+
+func TestStoreMultiList(t *testing.T) {
+	reexec.Init()
+
+	store := newTestStore(t, StoreOptions{
+		GraphDriverName: "vfs",
```

Comment:
Using the `vfs` driver is also common to the configuration used to run tests, and should be shared.